CHANGELOG.md

```diff
@@ -1,14 +1,14 @@
 # NEXT RELEASE
 
 ### Enhancements
-* None.
+* Add support for Google openId
 
 ### Fixed
 * <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-core/issues/????), since v?.?.?)
 * None.
  
 ### Breaking changes
-* None.
```

Comment:
Is this a breaking change?

Reply:
It technically breaks the API that the SDKs consume. But it's not actually breaking as far as our users are concerned.

How does core normally handle this?

Reply:
We normally only include user facing breaking changes here that's expected to bubble up to SDKs as breaking changes.

Reply:
Then I will remove this as a breaking change.

Reply:
In that case you should remove it from the `Breaking changes` section and leave it in `Enhancements`.